var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -246,6 +248,8 @@ def configure_args(self):
             "--enable-threads={0}".format(spec.variants['threads'].value),
             "--with-ch3-rank-bits={0}".format(
                 spec.variants['ch3_rank_bits'].value),
+            '--enable-wrapper-rpath={0}'.format('no' if '~wrapperrpath' in
```

Comment:
Aren't we missing the declaration of the `wrapperrpath` variant? Or am I overlooking something?

Reply:
Thanks a lot for catching it. I've fixed it.

```
commit f6eddf21effa6fb1c938e8d4176883958cb56408
Author: Hari Subramoni <subramoni.1@osu.edu>
Date:   Tue Oct 29 11:40:18 2019 -0400

    Add declaration of the wrapperpath variant.
        - Thanks to Massimiliano Culpo for the comment
```